### ISSUE TITLE
How should climbing gear work?

### ISSUE THREAD

**Issue by 0ffz** [MineInAbyss/StaminaClimb#10]
The options I've thought of so far:
- Limit climbing to a specific climbing pickaxe that can be crafted with a custom recipe
- Allow climbing with bare hands but using different tools could decrease the drain speed and length you can travel or allow for climbing on different blocks

**Comment by Mwr247** [comment]
The second option has the most appeal, IMO.
- Allows for a variety of potential upgrades, as you mentioned.
- Avoids requiring a specialty item using up a slot for a single purpose (reduces hassle of handling).
- In Adventure, where blocks can't be broken, having at least minimal climbing ability would come in handy if trapped in a shallow hole with no climbing tool.

As far as the idea of designing a custom tool, why not instead utilize the pickaxe specifically for this purpose?
- In Adventure, pickaxes won't be very useful for mining, but have many varieties/enchantments that could be tapped into to use for climbing instead.
- In Survival, it's one less item needed to haul around in the hotbar or inventory.
- For both, much more intuitive than having to learn a whole new recipe as a newcomer.
- If possible, could even tap into it's durability, decreasing as it's utilized for climbing.

**Comment by 0ffz** [comment]
Yeah I think eventually we'll have some geary items that drain stamina slower and stuff like that, it'll be super easy to do once that's done.
